Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM node:9
 # Create app directory
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
+# set default node environment
+ENV NODE_ENV production
```

Comment:
That's very dangerous in my opinion. The default should always be the closest to development. I'd leave empty as it was.

Reply:
i understand.  i do like to use the convention that the dockerfile documents the env variables that are configurable.  I just picked the wrong default.  should i delete or change value to 'development'?

Reply:
You can safely leave it empty or use "development" or just remove this line. Up to you :)

Reply:
done.  thanks for the help.